src/ui/SessionExpiryTracker/SessionExpiryTracker.tsx

```diff
@@ -64,19 +65,29 @@ const SessionExpiryTracker: React.FC = () => {
     }
   }, [sessionExpiryTimeString, checkSession, setupSessionIntervalCheck])
 
+  useEffect(() => {
+    if (!redirectToLogout) {
+      return
+    }
+
+    const handleLogout = async () => {
+      await fetch(`${config.API_URL}/logout`, {
+        method: 'POST',
+        credentials: 'include',
+      })
+      Cookies.remove(COOKIE_SESSION_EXPIRY)
+      localStorage.setItem(LOCAL_STORAGE_SESSION_EXPIRED_KEY, 'true')
+      localStorage.removeItem(LOCAL_STORAGE_SESSION_TRACKING_KEY)
+    }
+
+    handleLogout()
+  }, [redirectToLogout])
+
   if (!sessionExpiryTimeString || !redirectToLogout) {
     return null
   }
 
-  Cookies.remove(COOKIE_SESSION_ID, { path: '' })
-  localStorage.setItem(LOCAL_STORAGE_SESSION_EXPIRED_KEY, 'true')
-  localStorage.removeItem(LOCAL_STORAGE_SESSION_TRACKING_KEY)
-
-  return config.IS_SELF_HOSTED ? (
-    <Redirect to={'/'} />
-  ) : (
-    <Redirect to={'/login'} />
-  )
```

Comment:
What was the reasoning behind removing the two redirects?

Reply:
Mentioned on slack, will post here as well:

The Root Cause (to me) looks to be the persistence of the `sessionid` cookie even after the user was allegedly "logged out." The user was getting pushed to the login page thanks to the redirect at the bottom, and was being "locked" there because of the presence of the `session-expiry` cookie forcing the user to "login" and refresh that `session-expiry` cookie.
Now that we're actually removing the `sessionid` cookie, we can also remove the `session-expiry` cookie, leaving the user in a "clean" state, so on their next login we add both cookies back and can avoid infinite redirects.

This redirect is superfluous from the root cause and can be removed to reduce complexity in this part of the code